### ISSUE TITLE
Porting TestExpectations to WPT Metadata

### ISSUE THREAD

**Issue by KyleJu** [web-platform-tests/wpt-metadata#481]
Per Dave Tapuska's suggestions, WPT Metadata should be a subset of TestExpectations, but with a focus on chrome-specific test failures. The triage information in TestExpectations should be ported continuously to WPT Metadata. 

For now, we can write a ad-hoc script to port the existing triage information to WPT metadata and think of an automated way going forward (e.g. through a bot).

**Comment by Hexcles** [comment]
> (Why is this issue in wpt.fyi not wpt-metadata?)

Good point. I've moved the issue to wpt-metadata. (Old issue links should continue to work.)
